### ISSUE TITLE
lib/Unicode: status in core-p7 branch

### ISSUE THREAD

**Issue by jkeenan** [atoomic/perl#48]
In this ticket we will track our core-p7 progress on tests in directory `lib/Unicode`.

As 2020-07-09 10:20 EDT, we are getting this output from `make test` for the files in this directory.

```
../lib/Unicode/UCD.t ............................................... ok
Variable "$gid" is not imported at ../lib/User/grent.t line 33.
Variable "@grent" is not imported at ../lib/User/grent.t line 35.
Variable "@grent" is not imported at ../lib/User/grent.t line 37.
Variable "@grent" is not imported at ../lib/User/grent.t line 39.
Global symbol "$gid" requires explicit package name (did you forget to declare "my $gid"?) at ../lib/User/grent.t line 33.
Global symbol "@grent" requires explicit package name (did you forget to declare "my @grent"?) at ../lib/User/grent.t line 35.
Global symbol "@grent" requires explicit package name (did you forget to declare "my @grent"?) at ../lib/User/grent.t line 37.
Global symbol "@grent" requires explicit package name (did you forget to declare "my @grent"?) at ../lib/User/grent.t line 39.
Execution of ../lib/User/grent.t aborted due to compilation errors.
# Looks like your test exited with 255 just after 1.

```
These results were generated on an unthreaded build on linux.  The results might differ significantly if run on a threaded build.  There have been subsequent commits to this branch which may have impacted these results.  YMMV.

**Comment by atoomic** [comment]
this is fixed by f3608e7c

**Comment by jkeenan** [comment]
> this is fixed by [f3608e7](https://github.com/atoomic/perl/commit/f3608e7cd8d0059d19bdeae0a80091a7182beb77)

Confirmed.  Thanks.
